[CCLEX-187]: Added Namespace to entity details view and Cluster details page (#1201)
Mirantis/lens-extension-cc: CHANGELOG.md

- When setting the Management Cluster URL in the "Add Management Cluster" view, only the necessary portion of the URL will be used in order to establish the sync.
  - For example, if the user provides a URL to a child cluster, like `https://mcc.com/projects/my-project/clusters/my-cluster`, instead of the management cluster, only the origin, `https://mcc.com` in this case, will be used in order to help them be more successful when using the extension since the marketing term is somewhat confusing.

> - Added "Namespace" to each entity details view, replaced "Kind" with "Namespace" in Cluster details page.

Reviewer: Since we haven't published the cluster page yet, only this is necessary:
```suggestion
- Added "Namespace" to each entity details view in the Catalog.
```

Author: Fixed
